polaris.shopify.com/src/data/foundations.json

```diff
@@ -32,7 +32,7 @@
       ]
     },
     "intro": "Our mission is to make commerce better for _everyone_. Building products for everyone means creating inclusive content.",
-    "section": "content"
+    "category": "content"
```

Comment:
Love the consolidation here. I was unsure why it's called `content` but this is fine for now.